wagtailio/core/models/pages.py

```diff
@@ -130,3 +138,165 @@ class ContentPage(Page, HeroMixin, SocialMediaMixin, CrossPageMixin):
         index.SearchField("intro"),
         index.SearchField("body"),
     ]
+
+
+class ShowcasePage(Page, SocialMediaMixin, CrossPageMixin):
+    template = "patterns/pages/showcase_page/showcase_page.html"
+    htmx_template = "patterns/pages/showcase_page/showcase_page_fragment.html"
+
+    parent_page_types = ["core.HomePage"]
+
+    introduction = models.TextField(
```

Comment:
q: should this be a limited `RichTextField`? if not, wouldn't a `models.CharField` suffice?

Reply:
I've not really been given any brief of how long this should be, I tend to prefer leaving this to editor control unless there's a reason to lock it down?

Reply:
if it is kept to `p, a, ul, li, link, document` that should cover most uses without needing special styling or handling for something like headings

Reply:
You got it :)